crates/telio-wg/src/wg.rs

```diff
@@ -1819,4 +1817,32 @@ pub mod tests {
             ));
         }
     }
+
+    #[rstest]
+    #[case(None, None, false)]
+    #[case(Some(0), None, false)]
+    #[case(None, Some(0), false)]
+    #[case(Some(0), Some(0), true)]
+    #[case(Some(0), Some(1), true)]
+    #[case(Some(1), Some(0), true)]
+    #[case(Some(0), Some(11), true)]
+    #[case(Some(11), Some(0), false)]
+    fn test_bytes_and_timestamps(
+        #[case] rx_sec_ago: Option<u64>,
+        #[case] tx_sec_ago: Option<u64>,
+        #[case] expect_link_up: bool,
+    ) {
+        let now = Instant::now();
+        let rx_ts = rx_sec_ago.and_then(|v| now.checked_sub(Duration::from_secs(v)));
+        let tx_ts = tx_sec_ago.and_then(|v| now.checked_sub(Duration::from_secs(v)));
+        let rtt = Duration::from_secs(0);
```

Comment:
Ideally `rtt` could also be extracted into the `rstest` above, so it would also be configurable :), it can be explicitly set to zero in existing testcases but additional ones with non-zero value would be nice to have covered and should be pretty easy :)